modules/neo4j/src/test/java/org/testcontainers/containers/Neo4jContainerTest.java

```diff
@@ -183,6 +183,32 @@ public void shouldAddConfigToEnvironment() {
             .containsEntry("NEO4J_dbms_tx__log_rotation_size", "42M");
     }
 
+    @Test
+    public void shouldConfigureSingleLabsPlugin() {
+        Neo4jContainer<?> neo4jContainer = new Neo4jContainer<>("neo4j:4.4")
+            .withLabsPlugins(Neo4jLabsPlugin.APOC);
+
+        // needs to get called explicitly for setup
+        neo4jContainer.configure();
+
+        assertThat(neo4jContainer.getEnvMap())
+            .containsEntry("NEO4JLABS_PLUGINS", "[\"apoc\"]");
+    }
+
+    @Test
+    public void shouldConfigureMultipleLabsPlugins() {
+        // configureLabsPlugins {
+        Neo4jContainer<?> neo4jContainer = new Neo4jContainer<>("neo4j:4.4")
+            .withLabsPlugins(Neo4jLabsPlugin.APOC, Neo4jLabsPlugin.BLOOM);
+        // }
+
+        // needs to get called explicitly for setup
+        neo4jContainer.configure();
+
+        assertThat(neo4jContainer.getEnvMap().get("NEO4JLABS_PLUGINS"))
+            .containsAnyOf("[\"apoc\",\"bloom\"]", "[\"bloom\",\"apoc\"]");
+    }
+
```

Comment:
these tests are not starting the container BUT I suggest we still wrap it with `try-with-resources`, just in case we change something and `configure` starts creating some resources

Reply:
Thanks for the input. Better safe than sorry.

Reply:
All container creations are now wrapped in try-with-resources.